clang/test/CIR/CodeGen/vbase.cpp

```diff
@@ -16,4 +16,7 @@ void ppp() { B b; }
 // CHECK:  cir.global linkonce_odr @_ZTT1B = #cir.const_array<[#cir.global_view<@_ZTV1B, [0 : i32, 0 : i32, 3 : i32]> : !cir.ptr<!u8i>]> : !cir.array<!cir.ptr<!u8i> x 1>
 // CHECK:  cir.global "private" external @_ZTVN10__cxxabiv121__vmi_class_type_infoE
 // CHECK:  cir.global linkonce_odr @_ZTS1B = #cir.const_array<"1B" : !cir.array<!s8i x 2>> : !cir.array<!s8i x 2>
-// CHECK:  cir.global constant external @_ZTI1B = #cir.typeinfo<{#cir.global_view<@_ZTVN10__cxxabiv121__vmi_class_type_infoE, [#cir.int<2> : !s64i]> : !cir.ptr<!u8i>, #cir.global_view<@_ZTS1B> : !cir.ptr<!u8i>}>
+// CHECK:  cir.global "private" external @_ZTVN10__cxxabiv117__class_type_infoE : !cir.ptr<!cir.ptr<!u8i>>
```

Comment:
These ABI bits are not obvious without context, can you add a brief description on how are these things supposed to work and what is getting tested? Example in `vtable-rtti.cpp`.